ai_edge_torch/generative/examples/stable_diffusion/util.py

```diff
@@ -44,9 +44,15 @@ def get_file_path(filename, url=None):
 
 def move_channel(image, to):
   if to == "first":
-    return image.permute(0, 3, 1, 2)  # (N, H, W, C) -> (N, C, H, W)
+    if isinstance(image, torch.Tensor):
```

Comment:
can you tell me why we are doing this?

Reply:
The util was created for torch.Tensor.  The update makes it compatible with both torch.Tensor and numpy.ndarray